README.md

```diff
@@ -2,180 +2,177 @@
 
 [![Netlify Status](https://api.netlify.com/api/v1/badges/be93b718-a6df-402a-b4a4-855ba186c97d/deploy-status)](https://app.netlify.com/sites/kubernetes-io-main-staging/deploys) [![GitHub release](https://img.shields.io/github/release/kubernetes/website.svg)](https://github.com/kubernetes/website/releases/latest)
 
-This repository contains the assets required to build the [Kubernetes website and documentation](https://kubernetes.io/). We're glad that you want to contribute!
-
-- [Contributing to the docs](#contributing-to-the-docs)
-- [Localization READMEs](#localization-readmes)
+Welcome! This repository contains the assets required to build the [Kubernetes website and documentation](https://kubernetes.io/). We're glad you're interested in contributing.
+
+## Table of Contents
+1. [Using This Repository](#using-this-repository)
+2. [Prerequisites](#prerequisites)
+3. [Running the Website](#running-the-website)
+4. [Building the API Reference Pages](#building-the-api-reference-pages)
+5. [Troubleshooting](#troubleshooting)
+6. [Get Involved with SIG Docs](#get-involved-with-sig-docs)
+7. [Contributing to the Docs](#contributing-to-the-docs)
+8. [Localization READMEs](#localization-readmes)
+9. [Code of Conduct](#code-of-conduct)
+10.[Thank You!](#thank-you)
+
+---
 
 ## Using this repository
 
-You can run the website locally using [Hugo (Extended version)](https://gohugo.io/), or you can run it in a container runtime. We strongly recommend using the container runtime, as it gives deployment consistency with the live website.
+Run the website locally using [Hugo (Extended version)](https://gohugo.io/), or in a container runtime (recommend for deployment consistency with the live site).
 
 ## Prerequisites
 
-To use this repository, you need the following installed locally:
+To use this repository, you'll need:
 
 - [npm](https://www.npmjs.com/)
 - [Go](https://go.dev/)
 - [Hugo (Extended version)](https://gohugo.io/)
 - A container runtime, like [Docker](https://www.docker.com/).
 
 > [!NOTE]
-Make sure to install the Hugo extended version specified by the `HUGO_VERSION` environment variable in the [`netlify.toml`](netlify.toml#L11) file.
+Ensure you install the Hugo extended version specified by the `HUGO_VERSION` environment variable in the [`netlify.toml`](netlify.toml#L11).
 
-Before you start, install the dependencies. Clone the repository and navigate to the directory:
+## Running the Website
 
-```bash
-git clone https://github.com/kubernetes/website.git
-cd website
-```
+**Initial Setup**
 
-The Kubernetes website uses the [Docsy Hugo theme](https://github.com/google/docsy#readme). Even if you plan to run the website in a container, we strongly recommend pulling in the submodule and other development dependencies by running the following:
+ 1. **Clone the Repository**:
```

Comment:
Why indent this line with a space?
Space is meaningful in markdown.

Reply:
I have made all the changes as you have suggested above . Thank you for your valuable feedback.